### DISCUSSION TITLE
[完成委托]以如下方式上传图片。

### DISCUSSION THREAD

**Discussion by lantainAirblue** [share121/inter-knot#4648] [教程]
![Screenshot_20240909_002522](https://github.com/user-attachments/assets/95cddbd1-b258-4628-b5f3-4405b1c72432)请查收

**Comment by Ammon9813** [comment]
点了没反应怎么办啊T_T

**Comment by dzqwe** [comment]
### 方法2   
1.找一个图床   
2.获得Markdown链接
3,粘贴到发帖框内  
4.点击"Comment"   
5.发帖成功

**Comment by kelivvds** [comment]
为什么不翻译是不想吗？华为手机双指长按屏幕是可以翻译的，而且浏览器也是可以翻译。
